### PR TITLE
#167131452: view modules 

### PR DESCRIPTION
**Pull request by chekwas88**
### What does this PR do?

* Create the Dashboard Page

### Description of Task to be completed?

* create module component
* create Dashboard component
* create  modules actions
* create a reducer for modules actions
* Write tests for the components and actions

### Any background context you want to provide?

N/A

### What are the relevant Github issues?

N/A

### Questions:

N/A
### Screenshot
image

### CODE REVIEW

File: .sass-lint.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ files:
   include: 'sass/**/*.s+(a|c)ss'
 
 rules:
+<<<<<<< HEAD
```
Comment (Tyak99) [inline .sass-lint.yml]: @chekwas88 There is a merge conflict on here. Kindly review and fix it

File: package.json [hunk 2]
```diff
@@ -56,9 +54,8 @@
     "react-router-redux": "^4.0.8",
     "reactstrap": "^8.0.0",
     "redux": "^4.0.1",
-    "redux-mock-store": "^1.5.3",
     "redux-thunk": "^2.3.0",
-    "sinon": "^7.3.2",
+    "regenerator-runtime": "^0.13.2",
```
Comment (Tyak99) [inline package.json:60]: @chekwas88 Please what does this package do? If it is for async function. The project has `@babel/plugin-transform-runtime`

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,11 @@
+<<<<<<< HEAD
```
Comment (Tyak99) [inline src/style.css]: @chekwas88 There is a merge conflict in this file

File: src/test/Dashboard/Dashboard.spec.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+// import { BrowserRouter as Router } from 'react-router-dom';
```
Comment (Tyak99) [inline src/test/Dashboard/Dashboard.spec.js]: Kindly remove unhelpful comments @chekwas88

File: src/components/Button/button.scss [hunk 5]
```diff
@@ -5,6 +5,11 @@ $button: #fff;
   background-color: $btn;
   border-radius: 4px;
   color: $button;
+}
+.b-button {
+  background-color: #f3994a;
```
Comment (fortunee) [inline src/components/Button/button.scss]: Why aren't using the `$btn: #f3994a;` variable here?

File: src/components/Button/button.scss [hunk 6]
```diff
@@ -15,4 +20,4 @@ $button: #fff;
   background-color: $btn;
   border-color: $btn;
   box-shadow: none;
-}
+}
```
Comment (fortunee) [inline src/components/Button/button.scss]: Newline at the end of this file

File: src/components/Card/card.scss [hunk 7]
```diff
@@ -1,14 +1,10 @@
...
   margin: auto;
-  width: 400px;
   z-index: 1000;
 
   @media screen and (max-width: 420px) {
-    width: 300px;
+      width: 300px;
   }
-}
+}
```
Comment (fortunee) [inline src/components/Card/card.scss]: Add a newline here

File: src/components/Button/button.scss (hunk 5 above)
Comment (chekwas88) [reply src/components/Button/button.scss]: I have implemented that.

File: src/test/components/Modules.spec.js [hunk 8]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+// import { BrowserRouter as Router } from 'react-router-dom';
```
Comment (fortunee) [inline src/test/components/Modules.spec.js]: Why do we have commented out code here? @chekwas88

File: src/views/Dashboard/Dashboard.scss [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+.d-links li:hover {
+  border: 2px solid $sidebar-links-color;
+  border-radius: 2px;
+}
+
+a.m-card-link {
+  color: #000;
+}
+
+
```
Comment (fortunee) [inline src/views/Dashboard/Dashboard.scss]: Get rid of the whitespaces here and leave a new line

File: src/views/Signup/SignUp.scss [hunk 10]
```diff
@@ -99,6 +99,12 @@ html {
   text-align: center;
 }
 
+.headsi {
```
Comment (fortunee) [inline src/views/Signup/SignUp.scss]: `.headsi` as a className doesn't make sense to me, please change this to something that make sense?

File: src/views/Signup/SignUp.scss [hunk 11]
```diff
@@ -99,6 +99,12 @@ html {
...
+  color: $htmlcolor;
```
Comment (fortunee) [inline src/views/Signup/SignUp.scss]: `$htmlcolor` is a very very bad name, please change this

File: src/views/Signup/SignUp.scss [hunk 12]
```diff
@@ -127,4 +133,4 @@ html {
   font-size: 12px;
   margin-bottom: 5%;
   margin-top: 0.8%;
-}
+}
```
Comment (fortunee) [inline src/views/Signup/SignUp.scss]: Add a new line here

File: store/actions/moduleActions.js [hunk 13]
```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (fortunee) [inline store/actions/moduleActions.js]: This is getting too much, please move it to our `eslintrc` config file

**Review by fortunee — CHANGES_REQUESTED** [review]
Good work! Chekwas, please review my comments and refactor accordingly. Thanks

**Review by fortunee — APPROVED** [review]
Great job @chekwas88 👍

**Pull request merged**